imagetest/fixtures.go

```diff
@@ -199,3 +201,21 @@ func (t *TestVM) AddAliasIPRanges(aliasIPRange, rangeName string) {
 		}
 	}
 }
+
+// AddTestUser generate ssh key pair and return public key.
+func (t *TestVM) AddTestUser() ([]byte, error) {
```

Comment:
should be named like 'addSSHKey' or something, the user isn't involved here at all

Reply:
also, does this function belong on TestVM? it doesn't do anything to the VM struct or the associated create-instances step

Reply:
move this to testworkflow as it's set key file source path on workflow